indra/newview/llvoavatar.cpp

```diff
@@ -8056,21 +8058,24 @@ bool LLVOAvatar::getIsCloud() const
 			);
 }
 
-void LLVOAvatar::updateRezzedStatusTimers(S32 rez_status)
+void LLVOAvatar::updateRezzedStatusTimers(ERezzedStatus rez_status)
 {
 	// State machine for rezzed status. Statuses are -1 on startup, 0
 	// = cloud, 1 = gray, 2 = downloading, 3 = full.
 	// Purpose is to collect time data for each it takes avatar to reach
 	// various loading landmarks: gray, textured (partial), textured fully.
 
+	auto next = [](ERezzedStatus status) -> ERezzedStatus { return (ERezzedStatus)(status + 1); };
+	auto incr = [&](ERezzedStatus& status) { status = next(status); };
+
```

Comment:
This is an extreme overkill. Just inherit the enum from S32 and it should start working with basic increment.

Reply:
😱